pyramid_swagger/tween.py

```diff
@@ -542,10 +542,10 @@ def validate_response(response, validator_map):
 
 
 def prepare_body(response):
-    # content_type and charset must both be set to access response.text
-    if response.content_type is None or response.charset is None:
+    # content_type must be set to access response.text
+    if not response.content_type:
```

Comment:
👍  perfect ... we're safe then! I wanted to make sure that we're not missing something that could break ``pyramid_swagger`` users.
**Thanks a lot for the feedback!**

PS. you're right ... In the comment I presented a possible scenario, while the test uses ``WebOb<1.7, pyramid<1.8.0`` (which is an other possible scenario 😄 )